react/features/recording/components/LiveStream/AbstractLiveStreamButton.js

```diff
@@ -119,14 +122,18 @@ export function _mapStateToProps(state: Object, ownProps: Props) {
         } = state['features/base/config'];
         const { features = {} } = getLocalParticipant(state);
 
-        visible = liveStreamingEnabled
-            && (!enableFeaturesBasedOnToken
-                || String(features.livestreaming) === 'true');
+        visible = liveStreamingEnabled;
+
+        if (enableFeaturesBasedOnToken) {
+            visible = visible && String(features.livestreaming) === 'true';
+            disabled = String(features.livestreaming) === 'disabled';
```

Comment:
If it is not defined it will not be visible. 
Hum, thanks for bringing it up I see know the PR is not correct.

The idea behind the disabled is that you have features enabled, but for some reason, the admin one day decides and disable that feature for the organization, then we want to behave as it is hidden, as that feature is not enabled just for the user.
I will re-think it and will update the PR.